assets/sass/06-components/navigation.scss

```
@@ -263,7 +263,7 @@
 				margin: 0;
 				min-width: max-content;
 				position: absolute;
-				top: 100%;
+				top: calc(100% + 2px);
```

```suggestion
				top: calc(100% + 3px);
```
Changing this to 3px solves the outline visibility issue in Chrome.